### ISSUE TITLE
Question: Get parent branch from TaskManager

### ISSUE THREAD

**Issue by sabat24** [zielu/GitToolBox#133]
You lately made a feature that allows to compare current branch with another selected branch. It works great and for me it's a great improvement.
I wonder how hard would it be to add an option to retreive parent branch from TaskManager?
You can retrieve BranchInfo via `LocalTask.getBranches(original = true)` and current task you can get via `TaskManager.getActiveTask()`
You should receive something like
```
[localHash=d5adedb56ed3fb0b0365b76c15c72022baca5f70,localBranch=refs/heads/fb_,remote=RepoStatusRemote[remoteTrackingBranch=<null>,parentBranch=refs/remotes/origin/release-7.0.00,parentHash=d5adedb56ed3fb0b0365b76c15c72022baca5f70]]
```

Sometimes I need to switch between "main" branches and changing the selected branch for long named branches is quite tricky, because each time you have to go to settings and type branch name.

**Comment by zielu** [comment]
I will probably replace tracked remote option with Automatic:
1. use task
2. fallback to tracked remote

**Comment by zielu** [comment]
Released on EAP channel in 191.0.12

**Comment by zielu** [comment]
Make auto mode default and as first option in drop-down

**Comment by zielu** [comment]
Released in 191.1.0
